play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -39,17 +39,17 @@ Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	2 Actions
 Range:		40m
-Effect:		10 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
+Effect:		10 % chance to miss. Deal 52 piercing damage. AP level 3.
+If the needle penetrates the targets armor and is still alive, apply a 
```

Comment:
I need to update the image, but the machine that I develop on doesn't handle images well. I'll update that in a non-Combat 2.0 branch, because it needs to be updated regardless of the status of Combat 2.0